open_rpc/tests/publicAPI.spec.ts

```diff
@@ -0,0 +1,253 @@
+import {
+  AddressFilter,
```

Comment:
there is also publicAPI.spec.ts in test/integration/publicAPI.spec.ts. Not sure it's exactly the same